backend/djangoindia/api/serializers/event.py

```diff
@@ -2,7 +2,16 @@
 
 from djangoindia.db.models.event import Event, EventRegistration,Sponsor,Sponsorship
 
-
+class SponsorDetailsSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=255)
+    type = serializers.CharField()
+    logo = serializers.ImageField()
+    url = serializers.URLField(allow_blank=True, allow_null=True)
+    
+class SponsorSerializer(serializers.Serializer):
+    sponsor_details = SponsorDetailsSerializer()
+    tier = serializers.CharField()
+    type = serializers.CharField()

```

Comment:
Why this addition? these classes are already present in this file .

Reply:
sorry I didn't do that , but at the time of discarding commits and merging , these commits are also got discarded. so it got merged again.

Reply:
It would be helpful for us if you could self review your PR before/after pushing the changes.